src/pages/design/color.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Color
+description: >
+  The Zendesk Garden color system is a set of purposeful colors designed with
+  brand personality, usability, and accessibility in mind. The system is
+  broken down into two core palettes: UI and brand.
+---
+
+## UI colors
+
+The UI colors are used when creating interface elements. Each color has been
+designed to meet accessibility requirements.
+
+### Primary colors
+
+Primary colors are used for the structure of interfaces, actionable items,
+and validation. Colors in the 600, 700, and 800 ranges have a [WCAG
```

Comment:
How do we feel about putting references to color names and ranges into `monospace`? I typically use code formatting when referring to them in slack and other places. cc @vsrixyz

Reply:
I'll defer to @vsrixyz on matters of the words.

Reply:
No strong opinion on this. I say go for it!